src/cargo/ops/fix.rs

```diff
@@ -333,20 +333,22 @@ to prevent this issue from happening.
     Ok(())
 }
 
-/// Entry point for `cargo` running as a proxy for `rustc`.
-///
-/// This is called every time `cargo` is run to check if it is in proxy mode.
+/// Provide the lock address when running in proxy mode
 ///
-/// Returns `false` if `fix` is not being run (not in proxy mode). Returns
-/// `true` if in `fix` proxy mode, and the fix was complete without any
+/// Returns `None` if `fix` is not being run (not in proxy mode). Returns
+/// `Some(...)` if in `fix` proxy mode, and the fix was complete without any
```

Comment:
Part of this comment is not true anymore. Perhaps move them to `fix_exec_rustc`?